src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -119,12 +125,13 @@ public CommandBase turnOffBrake() {
         return new InstantCommand(() -> elevatorBrake.set(true), this);
     }
 
-    public CommandBase moveTicks(double ticks, double speed) {
+    public CommandBase moveDisengage(double ticks, double speed) {
```

Comment:
`moveForDisengage` would be better. We can probably also remove the ticks and speed arguments here since this is so specialized.